src/robusta/core/model/jobs.py

```diff
@@ -0,0 +1,142 @@
+from ...core.discovery import discovery
+
+SERVICE_TYPE_JOB = "Job"
+
+
+class JobContainer(BaseModel):
+    image: str
+    cpu_req: float
+    cpu_limit: float
+    mem_req: int
+    mem_limit: int
+
+    @staticmethod
+    def from_api_server(container: V1Container) -> "JobContainer":
+        requests: ContainerResources = discovery.container_resources(container, ResourceAttributes.requests)
+        limits: ContainerResources = discovery.container_resources(container, ResourceAttributes.limits)
+        return JobContainer(
+            image=container.image,
+            cpu_req=requests.cpu,
+            cpu_limit=limits.cpu,
+            mem_req=requests.memory,
+            mem_limit=limits.memory,
+        )
+
+
+class JobCondition(BaseModel):
+    type: str
+    message: Optional[str]
+
+
+class JobStatus(BaseModel):
+    active: int = 0
+    failed: int = 0
+    succeeded: int = 0
+    completion_time: Optional[str]
+    conditions: List[JobCondition]
+
+    @staticmethod
+    def from_api_server(job: V1Job) -> "JobStatus":
+        job_status: V1JobStatus = job.status
+        job_conditions: List[JobCondition] = [
+            JobCondition(type=condition.type, message=condition.message) for condition in (job_status.conditions or [])
+        ]
+        return JobStatus(
+            active=job_status.active or 0,
+            failed=job_status.failed or 0,
+            succeeded=job_status.succeeded or 0,
+            completion_time=str(job_status.completion_time),
+            conditions=job_conditions,
+        )
+
+
+class JobData(BaseModel):
+    backoff_limit: int
+    tolerations: Optional[List[Dict]]
+    node_selector: Optional[Dict]
+    labels: Optional[Dict[str, str]]
+    containers: List[JobContainer]
+    pods: Optional[List[str]]
+
+    @staticmethod
+    def from_api_server(job: V1Job, pods: List[str]) -> "JobData":
+        job_spec: V1JobSpec = job.spec
+        pod_spec: V1PodSpec = job_spec.template.spec
+        pod_containers: List[JobContainer] = [
+            JobContainer.from_api_server(container) for container in pod_spec.containers
+        ]
+        return JobData(
+            backoff_limit=job_spec.backoff_limit,
+            tolerations=[toleration.to_dict() for toleration in (pod_spec.tolerations or [])],
+            node_selector=pod_spec.node_selector,
+            labels=job.metadata.labels,
+            containers=pod_containers,
+            pods=pods
+        )
+
+
+class JobInfo(BaseModel):
+    name: str
+    namespace: str
+    type: str = SERVICE_TYPE_JOB
+    created_at: str
+    deleted: bool = False
+    cpu_req: float
+    mem_req: int
+    completions: int
+    status: JobStatus
+    job_data: JobData
+
+    def get_service_key(self) -> str:
+        return f"{self.namespace}/{SERVICE_TYPE_JOB}/{self.name}"
+
+    def __eq__(self, other):
+        if not isinstance(other, JobInfo):
+            return NotImplemented
+
+        return (  # ignore created_at because of dates format
```

Comment:
why should date formatting be different in the same robusta instance?  or it is because of db row?